### DISCUSSION TITLE
Protein-level FDR estimation with 2-species library

### DISCUSSION THREAD

**Discussion by ghost** [vdemichev/DiaNN#324] [General]
I'm using DIANN with a 2-species spectral library where one species is used as a decoy to estimate FDR as recommended in the paper. When I use the FDR calculation correction for the number of precursors in the spectral library (multiply by number of targets and divide by number of decoys) I get a fairly reasonable answer when doing precursor ID compared to what DIANN reports. However, when doing protein identification and multiply my FDR estimate by the number of target proteins and divide by the number of decoys it severely drops the number of identifications from what DIANN is reporting. Am I doing something incorrect and if so what is the right approach?

**Comment by ghost** [comment]
I should mention that the way I'm doing protein FDR is I take the maximum precursor associated with each target protein and each decoy protein and make that the protein score, which I believe is the conservative approach mentioned in the paper.

**Reply by vdemichev** [reply]
Hi, please rely on DIA-NN's PG.Q.Value or Lib.PG.Q.Value or Global.PG.Q.Value (please see the docs on details on these), to calculate the protein FDR. Taking the highest scoring precursor will not work well. Also, DIA-NN is not reproting all the decoys, so cannot do this for decoys.

Best,
Vadim
